MapboxNavigation.podspec

```diff
@@ -46,7 +46,7 @@ Pod::Spec.new do |s|
   s.dependency "MapboxCoreNavigation", "#{s.version.to_s}"
   s.dependency "MapboxMaps", "~> 10.7.0"
   s.dependency "Solar-dev", "~> 3.0"
-  s.dependency "MapboxSpeech", "~> 2.0"
+  s.dependency "MapboxSpeech", "~> 2.0.0"
```

Comment:
Is the intention to avoid 2.1 etc.? If so, that should be reflected in the Package.swift too.

Reply:
Good catch. Yes, idea is to avoid v2.1.x of Mapbox Speech because it drops iOS 11 support. I guess I'll do the same for the Mapbox Geocoder.